### DISCUSSION TITLE
Since java script is blocked can someone please Make the bookmarklets into html

### DISCUSSION THREAD

**Discussion by MilkMaker23** [3kh0/ext-remover#625] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I do not know how to convert js to html so can someone do it so i can use it to unblock my Chromebook since bookmarklet are blocked

**Comment by BroJac5246** [comment, marked as answer]
How would that work? You can only run JavaScript with bookmarklets, and any HTML would literally just contain embedded JavaScript. HTML is not a programming language, it's a markdown language.

**Reply by MilkMaker23** [reply]
it would work since at Javascript is blocked but i would be able to use a html file to unblock it

**Reply by 3s7vn** [reply]
how can you use an html file to unblock, js is blocked for me too

**Reply by NightScripts-API** [reply]
> it would work since at Javascript is blocked but i would be able to use a html file to unblock it

This simply would NOT work Javascript is ran through the System Browser as a Bookmarklet so the bookmarklet is ran as a URL into the browser which has it supported for Javascript, Chrome supports the Ability to use Javascript as a URL which creates a Bookmarklet. To create an HTML file it would not be possible to run Javascript through it, This is because your opening the file within the System Files Explorer and not the Browser, The File Explorer does not support it, Now even if you create a website with HTML, It will not be able to replicate a URL that isn't Blocked to Javascript unless you used a .href atrribute which would work but your school blocks the Javascript by URL, The only way Javascript is usable is by the URL, So this is SIMPLY not Possible.

**Reply by BroJac5246** [reply]
> it would work since at Javascript is blocked but i would be able to use a html file to unblock it

No it wouldn't. JavaScript is blocked. So if we put it in an HTML file, will it be unblocked? No, because it's still JavaScript, and the HTML file just tells the browser to run JavaScript.

Besides, if you click an HTML file that's a bookmark, it'll display the webpage that the HTML codes for-- it won't run on the current page, because how would that work?

**Reply by TheGooferFox** [reply]
Well im just confused how could javascript be blocked since mainly every website requires it.. They are wrong yes because well that is basically impossible on what their asking for unless someone makes an embedded  website URL blah blah blah that can run it etc etc new bypasses but overall javascript cannot be completely blocked for chrome OS and if it can be ig I'm just stupid lol

**Reply by BroJac5246** [reply]
> Well im just confused how could javascript be blocked since mainly every website requires it.. They are wrong yes because well that is basically impossible on what their asking for unless someone makes an embedded website URL blah blah blah that can run it etc etc new bypasses but overall javascript cannot be completely blocked for chrome OS and if it can be ig I'm just stupid lol

It appears that they just have bookmarklets blocked and can still use regular JavaScript. However, the above still applies.

**Reply by MilkMaker23** [reply]
[Untitled_ Apr 28, 2023 11_58 AM.webm](https://user-images.githubusercontent.com/96270326/235208557-8bd9701e-341d-4a4f-9d33-63c4e95b53c6.webm)

**Reply by Aka-but-nice** [reply]
Yes! An actual person with intellect. One site cannot interact with the site it opens due to CORS.

**Reply by mumbojum2** [reply]
cant u use cors-anywhere or am I just stupid

**Reply by Mazedotexe** [reply]
yes, and I'm sure changing it to something like /webstoreb would make it not blocked, but it still wouldn't work BECAUSE of CORS. the problem in the video is not CORS but even if it's solved CORS is a bigger problem that can't be solved.

**Comment by Kazuma-Itsuchi** [comment]
Try this. The posts OP listed it as a possible bypass for blocked bookmarklets. I can't test it tho bc my javascript never got blocked.

https://drive.google.com/file/d/1FGaGkP8sA87JdFMoxgjMm00Kpwh23ccV/view

**Reply by mllef1** [reply]
Why did you pack it?

**Reply by AshtonDavies** [reply]
The thing about executing eval() into a window like that is that it has to be done before the page is defined as the webstore, so that file is no use for bypassing bookmarklet blocking unless you manage to make the HTML page insert the script after the page is defined. I don't know my stuff when it comes to XSS, but maybe there's a trick to add to that eval script to fool the browser.

**Reply by AshtonDavies** [reply]
This does not work. It runs and displays the prompt before the page is defined as the webstore.

**Reply by akabutnicer** [reply]
This **will not work**. It's quite frankly impossible to run code on a different origin which was opened. Rather an XSS would work, although that is not what this "exploit" (which isn't one at one) is doing.

**Comment by ILikeBypassing** [comment]
why cant you just run the javascript code through console 🤷

**Reply by BroJac5246** [reply]
If their admins were smart enough to block JavaScript, then do you think they would be dumb enough to let students use Developer Tools? Probably not.

**Reply by ILikeBypassing** [reply]
you never know

**Reply by BroJac5246** [reply]
> you never know

You asked "why can't," not "can you," and I gave the correct answer for your question.

**Reply by PSYclownYT** [reply]
Um WhEl AcKtUaWwY 🤓🤓🤓 , they could have a cs or IT class, where you would need access to the console. that doesn't require access to bookmarklets.

**Reply by mumbojum2** [reply]
lol my school does but on any error page on the chrome web store it doesn't let you view or edit the html

**Comment by PSYclownYT** [comment]


**Reply by BroJac5246** [reply]
As I mentioned earlier, why would they have access to the console? Their admins were clearly smart enough to disable JavaScript.

**Reply by Hilololsusy** [reply]
At my school they disabled javascript without disabling inspect, I had to spread to the hwole school ltbeef inspect for them to remove it.

**Comment by PSYclownYT** [comment]
You can still just drag it into console to run the code. If console is blocked, then you can't.

**Comment by arnav-canada** [comment]
i know how to convert into html i just dont know how to run it with out installing anything because there might not be space on the cromebook

**Comment by hunteraugustgabriel** [comment]
you guys obviously don't go to an lausd school. They block literally everything.

**Comment by 3s7vn** [comment]
left a comment on here when i was in middle school, i'm a junior now. there's no way people are still trying to unblock games and stuff 😭😭

**Reply by cool-dacool** [reply]
lmao

**Reply by t-littlebits** [reply]
lol
